Tests/AutoCollection/HttpDependencyParser.tests.ts

```diff
@@ -44,6 +44,36 @@ describe("AutoCollection/HttpDependencyParser", () => {
             assert.equal(dependencyTelemetry.data, "http://bing.com:123/search");
             assert.equal(dependencyTelemetry.target, "bing.com:123 | abcdefg");
         });
+        
+        it("should return correct data for a URL without a protocol (https)", () => {
+            (<any>request)["method"] = "GET";
+            let parser = new HttpDependencyParser("a.bing.com:443/search", request);
+
+            response.statusCode = 200;
+            parser.onResponse(response);
+
+            let dependencyTelemetry = parser.getDependencyTelemetry();
+            assert.equal(dependencyTelemetry.dependencyTypeName, Contracts.RemoteDependencyDataConstants.TYPE_HTTP);
```

Comment:
Should this type be "TYPE_HTTPS"?

Reply:
There is currently no distinction in this field between http/s. The other option for the field, `TYPE_AI` means that the RDD has an appId correlated with it.